public/components/trace_analytics/components/traces/trace_view.tsx

```diff
@@ -250,6 +256,37 @@ export function TraceView(props: TraceViewProps) {
     );
   };
 
+  const TracetoLogsButton = () => {
+    return useMemo(
+      () =>
+        mode === 'data_prepper' || mode === 'custom_data_prepper' ? (
```

Comment:
should `mode` be a dependency?

Reply:
yes, updated [3de427e](https://github.com/opensearch-project/dashboards-observability/pull/2398/commits/3de427e9fe08191d804ca0496fec4e718e398b32)